diff --git a/openvdb/openvdb/thread/Threading.h b/openvdb/openvdb/thread/Threading.h
index b339a85ecc..f1433341f9 100644
--- a/openvdb/openvdb/thread/Threading.h
+++ b/openvdb/openvdb/thread/Threading.h
@@ -14,14 +14,16 @@
 #endif
 
 
-/// @note tbb/blocked_range.h is the ONLY include that persists from TBB 2020
-///   to TBB 2021 that itself includes the TBB specific version header files.
-///   In TBB 2020, the version header was called tbb/stddef.h. In 2021, it's
-///   called tbb/version.h. We include tbb/blocked_range.h here to indirectly
-///   access the version defines in a consistent way so that downstream
-///   software doesn't need to provide compile time defines.
 #include <tbb/blocked_range.h>
 #include <tbb/task.h>
+/// @note tbb/task_arena.h is the ONLY include that persists from TBB 2020
+///   to TBB 2021 to TBB 2023 that itself includes the TBB specific version
+///   header files.
+///   In TBB 2020, the version header was called tbb/stddef.h. In 2021+, it's
+///   called tbb/version.h. We include tbb/task_arena.h here to indirectly
+///   access the version defines in a consistent way so that downstream
+///   software doesn't need to provide compile time defines.
+#include <tbb/task_arena.h>
 #include <tbb/task_group.h>
 
 namespace openvdb {
